WMIAdventure/backend/WMIAdventure_backend/battle/businesslogic/BattleCard.py

```diff
@@ -10,6 +12,7 @@ class BattleCard:
     """
 
     """
+    states: List[CardState]
```

Comment:
Rozumiem edge case - karta jest blokowana dwa razy.
Natomiast implementacja tego mnie trochę zdziwiła, bo to dziwne żeby karta mogła być w kilku stanach jednocześnie.
Może ta mechanika stanów powinna być trochę inaczej zaimplementowana?

Reply:
Tutaj też widzę błąd - jeżeli karta zostanie zablokowana dwa razy (jakimś cudem), to dla obu tych bloków tyknie timer i koniec końców karta będzie zablokowana tylko raz.

Reply:
Faktycznie, powinna to być np. kolejka, a nie lista